src/Entities/Frames.jl

```diff
@@ -38,9 +38,12 @@ struct Frame{dim, T <: Real, N <: AbstractNode{dim, T},
         new{dim, T, N, VN, G}(nodes, cross_section, mass_matrix, Symbol(label))
     end
 end
-function Frame(n₁::N, n₂::N, g::G, mass_matrix::MassMatrix = Consistent,
-        label::Label = NO_LABEL) where {N <: AbstractNode, G <: AbstractCrossSection}
-    Frame(SVector(n₁, n₂), g, mass_matrix, label)
+function Frame(n₁::N1, n₂::N2, g::G, mass_matrix::MassMatrix = Consistent,
```

Comment:
Minor: Can you add a test in `frame.jl` with this case just creating single element ?